marketplaceItems/appreciate/appResources/appData/resources/appreciate_ui.html

```diff
@@ -0,0 +1,73 @@
+<!DOCTYPE html>
+<!--
+    Appreciate
+    Created by Zach Fox on 2019-01-30
+    Copyright 2019 High Fidelity, Inc.
+
+    Distributed under the Apache License, Version 2.0.
+    See the accompanying file LICENSE or http://www.apache.org/licenses/LICENSE-2.0.html
+-->
+
+<html>
+    <head>
+        <title>Appreciate</title>
+        <meta charset="utf-8">
+        <link rel="stylesheet" type="text/css" href="css/style.css">
+        <link href="https://fonts.googleapis.com/css?family=Raleway" rel="stylesheet">
+        <!--I modified jscolor to prevent it from printing the color code to the value of the input onchange. -->
+        <script type="text/javascript" src="js/jscolor.js"></script>
+    </head>
+
+    <body>
+        <div id="loadingContainer">
+        </div>
+
+        <div id="mainContainer">
+            <div id="titleBarContainer">
+                <div id="titleText" style="font-weight:bold;">
+                    Appreciate v1.0
+                </div>
+                <label class="switch">
+                    <input id="appreciateSwitch" type="checkbox" onclick="appreciateSwitchClicked(this)">
+                    <span class="slider round"></span>
+                </label>
+            </div>
+
+            <div id="firstRun" style="display:none;">
+                <i id="tutorialArrow"></i>
+                <div>&nbsp;Use this switch to enable and disable the Appreciate app.</div>
+            </div>
+
+            <div id="progressBarContainer">
+                <span style="font-weight:bold;">Intensity Meter</span>
+                <div id="currentIntensityDisplay">
+                    <span id="currentIntensity" style="width:0%"></span>
+                </div>
+                <div id="crosshatch"></div>
+            </div>
+
+            <label id="neverWhistleContainer">Never Whistle
```

Comment:
small thing, but can the Never Whistle go on the next line?

Reply:
yuuup!